plugin/src/main/cpp/extensions/openxr_fb_body_tracking_extension_wrapper.cpp

```diff
@@ -314,17 +314,22 @@ void OpenXRFbBodyTrackingExtensionWrapper::_on_process() {
 	//   with how models are designed, rather than the Meta positions of
 	//   the tips of the clavicles.
 	if (locations.isActive) {
+		// Construct a root under the hips pointing forwards
+		// IE, +z aligns with hips & user's real-world forward.
+		// (Remaining, however, on the XRorigin / Global XZ plane; root's basis rotated around Y to fit)
```

Comment:
This comment is great! However, I think we could remove all the other comments added in this PR - they don't really add much that isn't already in this comment, or clear from the line of code they are commenting.

Reply:
Thanks @dsnopek . But would you believe it?... Was aiming to update the PR this evening (UK time) after an afternoon working on my project itself...  only to discover during that afternoon session that - as I see it - the initial assertion "The Root joint carries no data, so instead place it on the ground under the hips pointing forwards" is erroneous. Or, more accurately, zeroing out the position of root seems inconsistent / illogical when the reference play area is set to LOCAL (aka XR_PLAY_AREA_SITTING).

The new Root orientation (pointing real-world forward, and parallel to real-world ground) is good, but I *believe* the original position should actually be maintained. This way, when switched to LOCAL, Root doesn't magically teleport up to Head, but maintains its location in tracker-space: being found where the ground would be, between the two feet. [Exactly where all the good stuff, avatar control, and skeleton modifications happen to be in my project!] 

A quick-and-dirty hack locally to comment out the 'slide', then rebuild, seems to confirm this indeed works better for all play area modes. Is that also your understanding of how Root should behave, and therefore the functionality I should tidy up and include in my new commit for review?

TL;DR - New Root orientation good. But origin should be copied over from the original Tracker itself?

Reply:
Perhaps. It'd be great if @Malcolmnixon could give his opinion - it's his original code that based the root position on the hips, rather than using the root tracker.